examples/cpp/SampleCppMini/main.cpp

```diff
@@ -31,7 +31,7 @@ void test_cpp_api(const char * token, int ticketType, const char *ticket)
 
     if (ticket != nullptr)
     {
-        const char *ticketNames[7] =
+        const char *ticketNames[8] =
```

Comment:
This is actually someone else's bug. Visual Studio 2022 claims that it's a gross violation to reserve smaller array than the number of items you're listing.. cleaning it up here.